src/main/resources/db/migration/V14_9__add_column_ubesvart_melding_ubehandlet.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE person_oversikt_status
+ADD COLUMN behandlerdialog_ubesvart_ubehandlet BOOLEAN DEFAULT FALSE;
+
+ALTER TABLE person_oversikt_status
+    RENAME COLUMN behandlerdialog_ubehandlet TO behandlerdialog_svar_ubehandlet;
```

Comment:
Her blir det altså både en ny kolonne, og rename av den gamle kolonnen for å gjøre de mer spesifikke til hhv ubesvart og svar

Reply:
Bra du husket indeksen 👍